cmd/wallet-web/src/router/routes.js

```diff
@@ -23,17 +23,12 @@ export default [
         meta: { requiresAuth: true },
       },
       {
-        path: 'logout',
```

Comment:
We don't need this path ?

Reply:
We shouldn't. It's a component, not a page. We are also using `Signout.vue` now, instead. Let me remove the `Logout.vue` as well